src/inc/utilcode.h

```diff
@@ -5110,7 +5110,7 @@ BOOL IsIPInModule(HMODULE_TGT hModule, PCODE ip);
 struct CoreClrCallbacks
 {
     typedef IExecutionEngine* (__stdcall * pfnIEE_t)();
-    typedef HRESULT (__stdcall * pfnGetCORSystemDirectory_t)(SString& pbuffer);
+    typedef HRESULT (STDAPICALLTYPE * pfnGetCORSystemDirectory_t)(SString& pbuffer);
```

Comment:
Yes, it would be nice to be consistent.

It may be best to just get rid of these callconv annotations from all internal methods and callbacks, and just use the system default for them.